layers/dhcpv4.go

```diff
@@ -0,0 +1,407 @@
+// Copyright 2012 Google, Inc. All rights reserved.
+// Copyright 2009-2011 Andreas Krennmair. All rights reserved.
```

Comment:
New files shouldn't have the Andreas copyright... that's just for files that originally came from https://github.com/akrennmair/gopcap :)